src/jenkins.py

```diff
@@ -68,12 +111,25 @@ def get_admin_credentials(connectable_container: Container) -> Credentials:
     """
     user = "admin"
     password_file_contents = str(
-        connectable_container.pull(JENKINS_PASSWORD_FILE_PATH, encoding="utf-8").read()
+        connectable_container.pull(PASSWORD_FILE_PATH, encoding="utf-8").read()
     )
     return Credentials(username=user, password=password_file_contents.strip())
 
 
-def calculate_env(admin_configured: bool) -> JenkinsEnvironmentMap:
+class EnvironmentMap(typing.TypedDict):
```

Comment:
perhaps just `Environment`? The type already indicates it is a dict?

Reply:
Fixed, thank you!